scio-core/src/main/scala/com/spotify/scio/values/SCollection.scala

```diff
@@ -308,15 +309,15 @@ sealed trait SCollection[T] extends PCollectionWrapper[T] {
    * this SCollection, and then flattening the results.
    * @group transform
    */
-  def flatMap[U: ClassTag](f: T => TraversableOnce[U]): SCollection[U] =
+  def flatMap[U: ClassTag](f: T => GenTraversableOnce[U]): SCollection[U] =
```

Comment:
What's the difference between `TO[U]` and `GTO[U]`? This would break backward compatibility so prefer not to change it.

Reply:
@nevillelyh well `TraversableOnce[+A] extends Any with GenTraversableOnce[A]`

probably missing coffee 😄 what do mean by breaking change?

Reply:
I mean libs with scio as "provided" dependency may get a `NoSuchMethod` error calling `flatMap(f: T => TO[U])`.